pkgs/by-name/ro/rosa/package.nix

```diff
@@ -22,10 +22,9 @@ buildGoModule rec {
   '';
 
   preCheck = ''
-    # Workaround for cmd/list/rhRegion/cmd_test.go:39
-    #   Failed to get OCM regions: Can't retrieve shards: Get "https://api.stage.openshift.com/static/ocm-shards.json": dial tcp: lookup api.stage.openshift.com on [::1]:53: read udp [::1]:55793->[::1]:53: read: connection refused
-    substituteInPlace "cmd/list/rhRegion/cmd_test.go" \
-      --replace-fail "TestRhRegionCommand" "SkipRhRegionCommand"
+    # remove tests that require network access
+    rm cmd/list/rhRegion/cmd_test.go
+    rm cmd/describe/cluster/cmd_test.go
```

Comment:
These should be skipped in the way described by the [Go chapter of the manual](https://nixos.org/manual/nixpkgs/stable/#ssec-skip-go-tests)

Reply:
Thank you for taking the time to point this out. I've reviewed the docs you linked as well as the [buildGoModule source](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/go/module.nix) and I think the refactored commit is better, including the change to use `excludedPackages` to exclude the `tests/e2e` test suite